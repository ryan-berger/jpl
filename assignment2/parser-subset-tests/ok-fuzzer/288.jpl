show K0Y_4j_(J4l_8x.8g_3K.)
print "#L"
assert L3I_(7, 4, .6533484), "x"
time read image "" to k
print "r"
