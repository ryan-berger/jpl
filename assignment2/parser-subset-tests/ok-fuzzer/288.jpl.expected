(ShowCmd (CallExpr K0Y_4j_ (VarExpr J4l_8x.8g_3K.)))
(PrintCmd "#L")
(StmtCmd
 (AssertStmt (CallExpr L3I_ (IntExpr 7) (IntExpr 4) (FloatExpr 0)) "x"))
(TimeCmd (ReadImageCmd "" (VarArgument k)))
(PrintCmd "r")
